index.dd

```diff
@@ -97,6 +97,24 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE
+----
+// Find the logarithm of every number
+// from 1 to 1_000_000 in parallel
+void main()
+{
+    import std.math, std.parallelism, std.stdio;
+
+    auto logs = new double[1_000_000];
+
```

Comment:
Might want to initialise it to iota. Also since the point is to parallelise it maybe add sone timing code to show that it runs faster?